### PR TITLE
Adding "orgánico argentina" organic label

### PR DESCRIPTION
**Pull request by mrmd8**
Per #4599 
Thanks

### CODE REVIEW

**Review by VaiTon — APPROVED** [review]

File: taxonomies/labels.txt [hunk 1]
```diff
@@ -15522,6 +15522,13 @@ fi:"Ekstra"-luokka
 fr:Catégorie “Extra”, catégorie supérieure, catégorie extra
 it:Classe &quot;Extra&quot;
 
+<en:Organic
+es:Orgánico Argentina
+image:es:Organico-argentina.67x90.png
```
Comment (stephanegigandet) [inline taxonomies/labels.txt:15527]:
```suggestion
image:es:organico-argentina.67x90.png
```

**Review by stephanegigandet — APPROVED** [review]
Thanks!

The image file also needs to be added to github, in the /html/images/lang/es/labels/ directory

**Comment by mrmd8** [comment]
> The image file also needs to be added to github, in the /html/images/lang/es/labels/ directory

Thanks @stephanegigandet , I though I had included it here with commit [c50c5ae](https://github.com/openfoodfacts/openfoodfacts-server/pull/4671/commits/c50c5aea6311993853961244b03fb133f1e70a23)

**Comment by stephanegigandet** [comment]
> > The image file also needs to be added to github, in the /html/images/lang/es/labels/ directory
> 
> Thanks @stephanegigandet , I though I had included it here with commit [c50c5ae](https://github.com/openfoodfacts/openfoodfacts-server/pull/4671/commits/c50c5aea6311993853961244b03fb133f1e70a23)

ah ok, my bad. just one thing: the file name needs to be in lowercase. I'll change it.

**Comment by mrmd8** [comment]
> just one thing: the file name needs to be in lowercase. I'll change it.

My bad, I won't make that mistake again. Sorry. 

Thanks!

**Pull request merged**